clang/lib/CIR/CodeGen/CIRGenCleanup.cpp

```diff
@@ -188,10 +188,20 @@ void EHScopeStack::popCleanup() {
   }
 }
 
+bool EHScopeStack::requiresLandingPad() const {
```

Comment:
We really shouldn't have a "landing pad" concept in CIR. That's an LLVM IR construct. Perhaps `requiresCatchOrCleanup`?